cli/morphir-elm-gen.js

```diff
@@ -20,6 +20,7 @@ program
     .option('-e, --target-version <version>', 'Language version to Generate.', '2.11')
     .option('-c, --copy-deps', 'Copy the dependencies used by the generated code to the output path.', false)
     .option('-m, --modules-to-include <comma.separated,list.of,module.names>', 'Limit the set of modules that will be included.')
+    .option('-s, --includeCodecs', 'Include codecs', false)
```

Comment:
The standard naming convention for options in commander is to use a dash between words so this should be `include-codecs`. This is important because the it affects how the options are mapped to properties.

Reply:
This has been fixed with the last push